dev/com.ibm.ws.jaxws.2.3.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -13,6 +13,12 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 
+webservice=JAX-WS web service configuration
+webservice.desc=Configuration for JAX-WS web services.
+
+portName=web service port name
```

Comment:
rewrite as:
portName=Web service port name
or
portName=JAX-WS web service port name

Reply:
Good catch! I'll make this change as well.